user_story/model/user_story.py

```diff
@@ -633,15 +633,35 @@ def _get_us_ca_numbers(self, cr, uid, ids, fieldname, arg,
             returns the state of user.story to which belong '''
         res = {}.fromkeys(ids)
         for ac in self.browse(cr, uid, ids, context=context):
-            ac_number = ac.name.split(')')
-            if len(ac_number) > 1:
-                ac_number = ac_number[0]
+            if ac.sequence_ac:
+                ac_number = str(ac.sequence_ac)
             else:
-                ac_number = ac.name[0:3]
+                ac_number = ac.name.split(')')
+                if len(ac_number) > 1:
+                    ac_number = ac_number[0]
+                else:
+                    ac_number = ac.name[0:3]
             res[ac.id] = \
                 'HU#' + str(ac.accep_crit_id.id) + ' CA#' + ac_number
         return res
 
+    def _get_default_sequence(self, cr, uid, context=None):
+        ''' Method to place the sequence of acceptability criteria '''
+        ac_ids = context.get('accep_crit_ids', [])
+        in_memory = [x for x in ac_ids if x[2]]
+        if in_memory:
+            order = sorted(in_memory, key=lambda x: x[2]['sequence_ac'])
+            seq = order[-1][2]['sequence_ac'] + 1
+        elif len(ac_ids):
+            order = [x[1] for x in ac_ids]
+            read_ac = self.read(cr, uid, order,
+                                ['sequence_ac'], context=context)
+            maxi = max(read_ac, key=lambda x: x['sequence_ac'])
+            seq = maxi['sequence_ac'] + 1
+        else:
+            seq = 1
+        return seq
+
```

Comment:
Yes.

The correct approach is:

```python
var=1
if something:
   var=logic()
if other:
   var=logic2()
```

Use:

```python
if something:
    var=logic()
elif other:
    var=logic2()
else:
    var=1
```
Is not considered clean.

Why?

The default behavior should be shown first and then the 1st case possible and then the second it is more readable than left the default value at the end in the else (yes I know python shit.!)